content/_changelogs/10.4.0.md

```diff
@@ -0,0 +1,32 @@
+## 10.4.0
+
+_Released 7/18/2022_
+
+**Features:**
+
+- The Cypress App's sidebar will now automatically collapse at narrow widths. Addressed in [#22393](https://github.com/cypress-io/cypress/issues/22393).
+
+**Bugfixes:**
+
+- Fixed an issue where file watchers were not completely closed prior to the Cypress App quitting, causing crashes on exit in some macOS environments. Fixes [#22026](https://github.com/cypress-io/cypress/issues/22026).
+- Fixed issues with reloading user configuration after it is updated. Fixes [#21785](https://github.com/cypress-io/cypress/issues/21785) and [#22389](https://github.com/cypress-io/cypress/issues/22389).
+- Fixed issues with `document.cookie` when testing multiple origins. Fixes [#22479](https://github.com/cypress-io/cypress/issues/22479) and [#22674](https://github.com/cypress-io/cypress/issues/22674).
+- Fixed an issue retrieving git information for non-bash Unix shells. Fixes [#22454](https://github.com/cypress-io/cypress/issues/22454).
+- Fixed an issue that prevented displayed "Last Updated" information on Windows for git repositories without a commit history. Addressed in [#22741](https://github.com/cypress-io/cypress/issues/22741).
+- Fixed styling of the Latest Runs tooltip to keep it centered over the latest run entry. Fixes [#22629](https://github.com/cypress-io/cypress/issues/22629) and [#22598](https://github.com/cypress-io/cypress/issues/22598).
+- Fixed styling issues with the Specs Explorer at narrow viewport widths. Fixes [#22626](https://github.com/cypress-io/cypress/issues/22626).
+- Fixed a visual issue in the reporter where special characters would be incorrectly formatted with markdown. Fixes [#5679](https://github.com/cypress-io/cypress/issues/5679).
+- Fixed an issue where the Proxy-Authorization header included capitalization that would fail with certain proxies. Fixes [#22383](https://github.com/cypress-io/cypress/issues/22383).
+- Added `react-dom` as a required dependency when setting up a React project for Component Testing. Fixes [#22371](https://github.com/cypress-io/cypress/issues/22371).
```

Comment:
Isn't this a breaking change to add a required dependency?

Reply:
This only affects the component testing setup wizard. Before it was lacking this dependency, which is required by React, not us.

Reply:
Gottcya, can we expand on this entry to provide more context?